db/migrate/20181003234523_initialize_stories_deleted_counts.rb

```diff
@@ -0,0 +1,18 @@
+class InitializeStoriesDeletedCounts < ActiveRecord::Migration[5.2]
+  def up
+    Keystore.transaction do
```

Comment:
```suggestion
    ActiveRecord::Base.connection.execute <<~SQL
      replace low_priority into keystores
      select concat("user:", users.id, ":stories_deleted"), 0
      from users left join stories on users.id = stories.user_id where stories.id is null;
      
      replace low_priority into keystores
      select concat("user:", users.id, ":stories_deleted"),  count(*)
      from users left join stories on users.id = stories.user_id
      where stories.is_expired = true group by users.id;
    SQL
```

Naming an ActiveRecord model in a migration is a setup for future frustration - if the model is removed or renamed, the migration can't run. I'll add the `good_migrations` gem to catch this, but for now here's a way to both avoid that and not only avoid (10k users * 3) round trips but make the database do all the work.

Please merge the other migration into this one to reuse this strategy.